elfo-core/src/actor_status.rs

```diff
@@ -17,10 +17,17 @@ pub struct ActorStatus {
 
 impl ActorStatus {
     pub const ALARMING: ActorStatus = ActorStatus::new(ActorStatusKind::Alarming);
+
+    #[cfg(not(feature = "test-util"))]
```

Comment:
maybe it will look better if we add two separate impl blocks for consts under different features?